### ISSUE TITLE
Looking for correct build instructions that produce nuget files

### ISSUE THREAD

**Issue by beeradmoore** [jamesmontemagno/StoreReviewPlugin#47]
## Feature Request:
TL;DR; What's the command to build this project and produce nuget packages?

Longer; 
Hey James, 
I am trying to get a different Xamarin+MAUI project to build nugets and I am getting all sorts of weird and wonderful errors. I thought the issue was project/csproj is setup incorrectly so I was going to re-factor some of its csproj info to look closer to what you have here as this seems very clean and simple.

However  when I download your project I am unable to build it, let alone attempt to package it. The command I am using to build is,

> msbuild StoreReview.Plugin.csproj -t:restore,build

but I get some weird and wonderful errors like 

> /Library/Frameworks/Mono.framework/Versions/6.12.0/lib/mono/msbuild/Current/bin/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.TargetFrameworkInference.targets(127,5): error NETSDK1045: The current .NET SDK does not support targeting .NET Core 6.0.  Either target .NET Core 5.0 or lower, or use a version of the .NET SDK that supports .NET Core 6.0. [/Users/bradmoore/Downloads/StoreReviewPlugin-master/StoreReviewPlugin-master/src/StoreReview.Plugin/StoreReview.Plugin.csproj]

Trying to build in VS4Mac I get a completely different issue.

> /Users/bradmoore/.nuget/packages/msbuild.sdk.extras/3.0.44/Build/Workarounds.targets(5,5): Error: If you are building projects that require targets from full MSBuild or MSBuildFrameworkToolsPath, you need to use desktop msbuild ('msbuild.exe') instead of 'dotnet build' or 'dotnet msbuild' (StoreReview.Plugin)

which I think means I have an issue with my system in general. Would you be able to provide your Azure Pipelines config (or at least the main build part) so I know that at least I have a working command and broken system so I can try get that to work and hopefully not just get building to work but creating of a nuget package.

I am not asking for help to fix my system, just want to remove some potential unknowns. Dropping system info below juuust incase you had any insight into the build issues (but again, if not no problemo)

```
Visual Studio Community 2022 for Mac
Version 17.6.3 (build 421)
Installation UUID: b3113656-7637-4408-8b1d-08bb97bf6871

Runtime
.NET 7.0.3 (64-bit)
Architecture: X64
Microsoft.macOS.Sdk 13.1.1007; git-rev-head:8afca776a0a96613dfb7200e0917bb57f9ed5583; git-branch:release/7.0.1xx-xcode14.2

Roslyn (Language Service)
4.6.0-3.23180.6+99e956e42697a6dd886d1e12478ea2b27cceacfa

NuGet
Version: 6.4.0.117

.NET SDK (x64)
SDK: /usr/local/share/dotnet/sdk/7.0.307/Sdks
SDK Versions:
	7.0.307
	7.0.306
	7.0.304
	7.0.302
	7.0.203
	7.0.202
	7.0.201
	7.0.200
	7.0.103
	7.0.102
	7.0.101
	7.0.100
	6.0.413
	6.0.412
	6.0.410
	6.0.408
	6.0.407
	6.0.406
	6.0.405
	6.0.404
	6.0.403
MSBuild SDKs: /Applications/Visual Studio.app/Contents/MonoBundle/MSBuild/Current/bin/Sdks

.NET Runtime (x64)
Runtime: /usr/local/share/dotnet/dotnet
Runtime Versions:
	7.0.10
	7.0.9
	7.0.7
	7.0.5
	7.0.4
	7.0.3
	7.0.2
	7.0.1
	7.0.0
	6.0.21
	6.0.20
	6.0.18
	6.0.16
	6.0.15
	6.0.14
	6.0.13
	6.0.12
	6.0.11
	5.0.17
	3.1.27

Xamarin.Profiler
Version: 1.8.0.49
Location: /Applications/Xamarin Profiler.app/Contents/MacOS/Xamarin Profiler

Updater
Version: 11

Apple Developer Tools
Xcode: 14.3 21812
Build: 14E222b

Xamarin.Mac
Version: 9.3.0.6 Visual Studio Community
Hash: 97731c92c
Branch: xcode14.3
Build date: 2023-04-11 22:38:35-0400

Xamarin.iOS
Version: 16.4.0.6 Visual Studio Community
Hash: 97731c92c
Branch: xcode14.3
Build date: 2023-04-11 22:38:36-0400

Xamarin Designer
Version: 17.6.3.9
Hash: 2648399ae8
Branch: remotes/origin/d17-6
Build date: 2023-08-02 19:04:23 UTC

Xamarin.Android
Version: 13.2.1.2 (Visual Studio Community)
Commit: xamarin-android/d17-5/a8a26c7
Android SDK: /Volumes/Storage/Library/Developer/Xamarin/android-sdk-macosx
	Supported Android versions:
		12.1 (API level 32)
		12.0 (API level 31)
		11.0 (API level 30)
		10.0 (API level 29)
		13.0 (API level 33)

SDK Command-line Tools Version: 7.0
SDK Platform Tools Version: 34.0.1
SDK Build Tools Version: 33.0.0

Build Information: 
Mono: d9a6e87
Java.Interop: xamarin/java.interop/d17-5@149d70fe
SQLite: xamarin/sqlite/3.40.1@68c69d8
Xamarin.Android Tools: xamarin/xamarin-android-tools/d17-5@ca1552d

Microsoft Build of OpenJDK
Java SDK: /Library/Java/JavaVirtualMachines/microsoft-11.jdk
11.0.16.1
Android Designer EPL code available here:
https://github.com/xamarin/AndroidDesigner.EPL

Eclipse Temurin JDK
Java SDK: /Library/Java/JavaVirtualMachines/temurin-8.jdk
1.8.0.302
Android Designer EPL code available here:
https://github.com/xamarin/AndroidDesigner.EPL

Android SDK Manager
Version: 17.6.0.50
Hash: a715dca
Branch: HEAD
Build date: 2023-08-02 19:04:28 UTC

Android Device Manager
Version: 0.0.0.1309
Hash: 06e3e77
Branch: HEAD
Build date: 2023-08-02 19:04:28 UTC

Build Information
Release ID: 1706030421
Git revision: c90350679f1b9295edc584519320b13d06489853
Build date: 2023-08-02 19:02:50+00
Build branch: release-17.6
Build lane: release-17.6

Operating System
Mac OS X 13.5.0
Darwin 22.6.0 Darwin Kernel Version 22.6.0
    Wed Jul  5 22:21:56 PDT 2023
    root:xnu-8796.141.3~6/RELEASE_X86_64 x86_64


```
